### PR TITLE
web-on-chain: improve and adjust for v27

### PR DESCRIPTION
**Pull request by Thykof**
- [x] add web on chain description in signature
- [x] increase default mas gas to 1 000 000 000 for web on chain to work on v27
- [x] rename some variables
- [x] keep same chunk size for web on chain

## Delivery Quality Checklist

- [ ] **Breaking Changes in API:**
  Does this PR introduce breaking changes in the API?
    - If yes, have you considered making it backward compatible?
    - If backward compatibility is not considered, set the "breaking-change" label.

- [ ] **Changelog:**
  - [ ] For bugfix PR, set the "bugfix" label
  - [ ] If this change should not appear in changelog, use "ignore-for-changelog" label

- [ ] **Version Update Handling:**
  Have you ensured that the version update by user is handled correctly?

- [ ] **PR Dependency:**
  Does this PR depend on another PR?
    - If yes, is it necessary for the dependency to be released prior to merging this one?

- [ ] **Documentation:**
  - [ ] Are any necessary changes made to user-facing documentation?
  - [ ] Confirm that API documentation is updated with any relevant changes.
  - [ ] Check that README and other documentation files are accurate and current.

### CODE REVIEW

**Comment by Thykof** [comment]
I tested, it doesn't work because of https://github.com/massalabs/station-massa-wallet/issues/845

but when I don't use correlation id (set correlation id manually in the code to "" in the /sign API call), it works for flappy and laby websites.

**Comment by Thykof** [comment]
tested on securenet, it works, for webonchain, better put some fees (1000nanoMAS) on the appendBytesToWebsite.

File: pkg/onchain/website/uploader.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 var content embed.FS
 
 const (
-	blockLength = 260000
+	blockLength = 78000
```
Comment (peterjah) [inline pkg/onchain/website/uploader.go:23]: why this value ? the new blockSize is 300kb
puting 78kb here will make website upload very painfull :/ (lot of chunks to upload)

**Review by peterjah — CHANGES_REQUESTED** [review]

File: pkg/onchain/website/uploader.go (hunk 1 above)
Comment (Thykof) [reply pkg/onchain/website/uploader.go:23]: https://github.com/massalabs/station/issues/1263#issuecomment-1806512678

Comment (Thykof) [reply pkg/onchain/website/uploader.go:23]: how to chose the value then?

**Review by thomas-senechal — APPROVED** [review]

**Pull request merged**